Icebnb/Icebnb/RadiusController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  RadiusController.swift
+//  Icebnb
+//
+//  Created by Robert Mogos on 12/10/2017.
+//  Copyright © 2017 Robert Mogos. All rights reserved.
+//
+
+import UIKit
+import KUIPopOver
+import InstantSearchCore
+import AlgoliaSearch
+
+class RadiusController: UIViewController {
+  @IBOutlet weak var slider: PRGRoundSlider!
+  let searcher: Searcher
```

Comment:
We're sure to have a searcher (or this component does not make sense). So you can force unwrap like you did for the slider

Reply:
No need for force unwrapping since it's initialised in the constructer